app/lib/services/sockets/transcription_connection.dart

```diff
@@ -310,7 +323,7 @@ class TranscripSegmentSocketService implements IPureSocketListener {
     final recordingsLanguage = SharedPreferencesUtil().recordingsLanguage;
     var params = '?language=$recordingsLanguage&sample_rate=$sampleRate&codec=$codec&uid=${SharedPreferencesUtil().uid}'
         '&include_speech_profile=$includeSpeechProfile&new_memory_watch=$newMemoryWatch&stt_service=${SharedPreferencesUtil().transcriptionModel}';
-    String url = '${Env.apiBaseUrl!.replaceAll('https', 'wss')}listen$params';
+    String url = '${Env.apiBaseUrl!.replaceAll('https', 'wss')}v2/listen$params';
```

Comment:
does speech profile still use v1? or go with v2

Reply:
Good question, it shouldn't matter really, as long as it returns the transcripts, but that's a good one, will check.